### DISCUSSION TITLE
Ban rotation

### DISCUSSION THREAD

**Discussion by 767339608** [fabricjs/fabric.js#6847] [Q&A]
How do I disable rotation in the selected selection

**Comment by asturur** [comment]
Please have a look at our controls api here:
http://fabricjs.com/controls-api
There are at least 2 way to do it. Read all the page and if is not clear we are going to update the  page.

**Reply by 767339608** [reply]
I want to disable rotation if there are at least 2 elements in the selection but the set property can only set a single disable rotation such as SetControlVisible or LockRotation

**Reply by asturur** [reply]
In understand from your answer the doc wasn't clear from the first read.

![image](https://user-images.githubusercontent.com/1194048/106439846-73684b00-6478-11eb-8e63-b3376a9d0417.png)

Look at this sentence from the DOC, the activeSelection is `fabric.ActiveSelection` and has its own property and so can host its control set without a rotation control.

Does it help understanding better the documentation?
